amqp/message_sender.go

```diff
@@ -85,12 +78,3 @@ func failOnError(err error, msg string) {
 		log.Fatalf("%s: %s", msg, err)
 	}
 }
-
```

Comment:
I was using the hasError method as it sets the body and status code when there's a problem with publishing to the MQ, but we can look at the logs for debugging issues with message sending.